ckanext/switzerland/commands.py

```diff
@@ -15,6 +17,10 @@ class OgdchCommand(ckan.lib.cli.CkanCommand):
         paster ogdch help
         # Cleanup datastore
         paster ogdch cleanup_datastore
+        # Cleanup harvester jobs and objects:
+        # - deletes the all harvest jobs and objects except the latest
+        # - how many jobs to keep can be set in the ckan config
+        paster ogdch cleanup_harvesters [{source_id}]
```

Comment:
I like this a lot that you can test it per source :)

Reply:
Yes, thank you.